src/openms/extern/CMakeLists.txt

```diff
@@ -38,7 +38,12 @@ endif()
 ##
 ## external packages (with fallback option to local version)
 ##
-option(USE_EXTERNAL_JSON "Use an external nlohmann::json library" OFF)
+if(PACKAGE_TYPE STREQUAL "deb")
```

Comment:
I probably wouldn't do this here? The cmake here shouldn't need to know about our packaging.

Reply:
Hm. I looked into adding this in package_deb.cmake, but that is included after the extern cmake, so setting USE_EXTERNAL_SQLITECPP there doesn't exclude the add_subdirectory's that were added in the extern CMakeLists. Right?